Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -6141,6 +6141,7 @@ static UITextAutocapitalizationType toUITextAutocapitalize(WebCore::Autocapitali
     case WebCore::AutofillFieldName::TelLocalSuffix:
     case WebCore::AutofillFieldName::TelExtension:
     case WebCore::AutofillFieldName::Impp:
+    case WebCore::AutofillFieldName::OneTimeCode:
```

Comment:
You can return `UITextContentTypeOneTimeCode` for this case.

Reply:
> You can return `UITextContentTypeOneTimeCode` for this case.

Thanks — I just now pushed an update with that change